src/askscript/__tests__/99-e2e-tests/e2e-12-bubble_sort.ask

```diff
@@ -0,0 +1,13 @@
+// Bubble sort
+ask {
+  let arr = [342.34, 35767, 2, 117, 999, 3435]
+  const len = arr:length
+  for (let i = 0; i < len; i = i + 1) {
+	  for (let j = 0; (j + 1) < len; j = j + 1){
+		  if (arr:at(j) > arr:at(j + 1)) {
+			  arr = arr:set(j, arr:at(j + 1)):set(j + 1, arr:at(j))
```

Comment:
@czerwinskilukasz1 look how this expression is evaluated by runtime 🙀. this is `swap(at[j], at[j+1]`

Reply:
@mhagmajer , cool.  
By the way, how about we implement a swap resource? :) 
https://github.com/xFAANG/askql/issues/213

Reply:
@czerwinskilukasz1 I guess what I meant was to show you that we actually have a working swap inliner in this language :)